modules/search/instant-search/components/search-results.jsx

```diff
@@ -36,28 +38,15 @@ class SearchResults extends Component {
 		return sprintf( _n( '%s result', '%s results', total, 'jetpack' ), num );
 	}
 
-	renderEmptyResults( { showText = false } = {} ) {
-		return (
-			<div className="jetpack-instant-search__search-results">
-				{ showText && (
-					<div>
-						<h3>{ sprintf( __( 'No Results.', 'jetpack' ), this.props.query ) }</h3>
-					</div>
-				) }
-			</div>
-		);
-	}
-
 	render() {
 		const { query } = this.props;
 		const { results = [], total = 0, corrected_query = false } = this.props.response;
 		const hasQuery = query !== '';
 		const hasCorrectedQuery = corrected_query !== false;
+		const hasResults = total > 0;
+
 		if ( ! hasQuery && ! hasFilter() ) {
-			return this.renderEmptyResults();
-		}
-		if ( total === 0 ) {
-			return this.renderEmptyResults( { showText: true } );
+			return <SearchResultsEmpty />;
```

Comment:
@jsnmoon can we just `return null` here instead or do we need the empty div to keep the portal happy?

Reply:
I think `return null` should be fine.